packages/service/tests/acceptance/test_logging.py

```diff
@@ -105,9 +105,12 @@ def test___streaming_data_measurement___measure___server_call_logged(
         )
         num_responses = 10
 
+        metadata = stub_v2.GetMetadata(v2_measurement_service_pb2.GetMetadataRequest())
```

Comment:
These tests have arrange/act/assert blocks separated by one blank line. Remove the blank line above metadata so it's part of the arrange block.
```suggestion
        )
        num_responses = 10
        metadata = stub_v2.GetMetadata(v2_measurement_service_pb2.GetMetadataRequest())
```